src/dji_osdk_ros/modules/vehicle_wrapper.cpp

```diff
@@ -2468,6 +2468,19 @@ bool VehicleWrapper::getHMSDeviceIndex(uint8_t& deviceIndex)
     }
   }
 
+  bool VehicleWrapper::changeH264Source(LiveView::LiveViewCameraPosition pos, LiveView::LiveViewCameraSource source)
+  {
+    if (!vehicle->advancedSensing)
+    {
+      return false;
```

Comment:
What about debugging msgs?

Reply:
The debug messages are in the service callback. This file is a pure C++ wrapper without ROS.